arbitrum-docs/getting-started-users.mdx

```diff
@@ -10,32 +10,95 @@ import PublicPreviewBannerPartial from './partials/_public-preview-banner-partia
+Now choose the desired network you wish to bridge to, e.g., Arbitrum One or Arbitrum Nova. Note that Arbitrum Goerli only appears if you are connected to the Goerli network rather than the Ethereum Mainnet.
+
+![Add the desired Arbitrum Network to your MetaMask wallet](assets/getting_started_users_2.png)
+
+Select the token you want to bridge in the token drop-down menu. You can also enable/disable the token lists by clicking `Manage Token lists` button on the bottom right corner of the drop-down menu.
+
+![Select the token to bridge](assets/getting_started_users_3.png)
+
+Enter the amount of ETH/ tokens you want to bridge over in the “Mainnet” field and then press `deposit`. Follow the prompts on MetaMask.
+
+![Enter the amount of tokens to bridge](assets/getting_started_users_4.png)
+
+After you submit the transaction through MetaMask you can expect your funds to arrive on Arbitrum One within roughly 10 minutes to an hour (depending on the chain congestion).
+
+Also make sure your MetaMask wallet is set to any of the Arbitrum One **or** Arbitrum Nova or **or** Arbitrum Goerli networks (depending on which one you chose to bridge to) so you can see your funds when they arrive.
+
+#### 3- Withdraw Your ETH/ Tokens (L2 -> L1)
+
+:::info There's At Least A 7 Day Withdrawal Period for Arbitrum One and Nova Networks
+Once you move your funds from Arbitrum One or Nova through the Arbitrum bridge, you will have to wait for at least 7 days to recieve them on Ethereum Mainnet.
+
+:::
+
+To move your funds back to Ethereum Mainnet, your need to be logged in to the [Arbitrum bridge](https://bridge.arbitrum.io/) with your wallet and make sure you are connected to he Arbitrum network you want to withdraw from (Arbitrum One, Arbitrum Nova, or Arbitrum Goerli).
+Select the token you want to bridge in the token drop-down menu. 
+
+![Select the token to withdraw](assets/getting_started_users_5.png)
+
+Enter the amount of ETH/ tokens you want to bridge over in the `L2` field and then press `withdraw`. Follow the prompts on MetaMask.
+
+_**Please make sure you leave enough ETH on your wallet to pay for the transaction, otherwise there will be no MetaMask popup.**_
 
-To move your Ether and Tokens between Arbitrum and Ethereum chains, visit [bridge.arbitrum.io](https://bridge.arbitrum.io/).
+![Enter the amount of token to withdraw](assets/getting_started_users_6.png)
 
-#### Use L2 Dapps!
+A countdown will pop up stating you’ll get your funds in 7-8 days.
 
-Interacting with Arbitrum chains will feel very similar to using Ethereum, just cheaper and faster! To get a sense of what's out there, you can check out our [portal page](https://portal.arbitrum.one/), where we showcase some of the dApps, wallets, and infrastructure currently live on Arbitrum One.
+You can check the status of your withdrawal by clicking on your profile on the top right and opening the `Transactions`, and claim it there when it’s ready.
```

Comment:
```suggestion
You can check the status of your withdrawal by clicking on your profile on the top right and opening the `Transactions` tab, and claim it there when it’s ready.
```

Reply:
And this one. You could also, not add "tab", and remove "the" before Transactions.
Whatever you prefer.

Reply:
Your suggested commit sounds good!